lib/src/component_declaration/component_base_2.dart

```diff
@@ -58,42 +108,92 @@ part of over_react.component_declaration.component_base;
 ///
 /// > Related: [UiStatefulComponent2]
 abstract class UiComponent2<TProps extends UiProps> extends react.Component2
-    with _DisposableManagerProxy
+    with DisposableManagerProxy, GeneratedClass
     implements UiComponent<TProps> {
+  // ***************************************************************************
```

Comment:
I felt the need to reorganize these members since they were all over the place...

Reply:
It will probably be easier to review this file by just looking at the new version